### ISSUE TITLE
Build fails without auto-install-peers=true in .npmrc

### ISSUE THREAD

**Issue by 5000164** [kuma-ui/kuma-ui#361]
I tried to clone the repository and run `pnpm install` and `pnpm build` as instructed in CONTRIBUTING.md, but the build failed.
I'm not familiar with pnpm, but after some research, I found that adding `auto-install-peers=true` to `.npmrc` made both pnpm install and pnpm build succeed.
Is this configuration necessary for development?
The versions I am using are nodejs 18.18.2 and pnpm 8.5.1.

**Comment by poteboy** [comment]
@5000164 
Hmm, it's strange. The default value for auto-install-peers is set to true, so explicitly specifying this value as true shouldn't change the behavior. It's just my guess, but since our packages depend on each other, perhaps running pnpm build multiple times resolvedd the internal mismatch?

**Comment by 5000164** [comment]
@poteboy 
Thank you for your response.
Investigation after reading your comment, I realized that although I thought I was using pnpm 8.5.1 in my local environment, the version that was actually running was 7.29.0.
Upon checking, I found that pnpm added the auto-install-peers configuration from version 7.1.3, and it became enabled by default from version 8.0.0. ref. https://github.com/pnpm/pnpm/releases/tag/v7.1.3 https://github.com/pnpm/pnpm/releases/tag/v8.0.0
There is no `auto-install-peers=true` setting in .npmrc seems like it is more appropriate for the engines configuration in package.json to specify `"pnpm": ">=8"`.
I have submitted a new PR reflecting this change. #363
